decidim-participatory_processes/app/commands/decidim/participatory_processes/create_space_page.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    class CreateSpacePage < Decidim::Command
+      # Public: Initializes the command.
+      #
+      # organization - A object with the params.
+      def initialize(organization)
+        @organization = organization
+      end
+
+      # Executes the command.
+      # i18n-tasks-use t('decidim.system.default_pages.terms-of-service')
```

Comment:
Doesn't make sense

Reply:
Hmmmz ... aparently i am a Ruby Interpreter myself ... i am not reading comments either :rofl:

Reply:
Fixed in [fc481b1](https://github.com/decidim/decidim/pull/15157/commits/fc481b1b82d693ff7f30765d923fa9703f8b696a)